geo-open/package.yaml

```diff
@@ -10,6 +10,21 @@ description: |
   database format. This package provides two contexts for IP enrichment:
   country-only lookups and combined country+ASN lookups.
 
+inputs:
+  enrich-input:
+    name: Enrichment input topic
+    description: |
+      The name of the input topic used for OCSF event enrichment.
+    type: string
+    default: ocsf
+
+  enrich-output:
+    name: Enrichment output topic
+    description: |
+      The name of the output topic used for OCSF event enrichment.
+    type: string
+    default: ocsf-geo-open
```

Comment:
Default name of the topic?

Reply:
Maybe no default and require the user to add one?

Reply:
I think that makes it a bit annoying to use if you don't want to use this feature.

Reply:
Fine, we can leave it as is.